docs/ruleset_functions/is_hvac_sys_fluid_loop_purchased_heating.md

```diff
@@ -0,0 +1,27 @@
+# is_hvac_sys_fluid_loop_purchased_heating  
+
+**Description:** Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+
+**Inputs:**  
+- **B-RMR**: To evaluate if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop.   
+- **hvac_b.id**: The id of the hvac system to evaluate.  
+
+**Returns:**  
+- **is_hvac_sys_fluid_loop_purchased_heating**: Returns TRUE if the fluid loop associated with the heating system associated with the HVAC system is attached to an external purchased heating loop. Returns FALSE if this is not the case.   
+ 
+**Function Call:** None  
+
+## Logic:   
+- Set is_hvac_sys_fluid_loop_purchased_heating equal to FALSE: `is_hvac_sys_fluid_loop_purchased_heating = FALSE`  
+- Check if RMR is not modeled with external fluid source: `if RMR.RuleModelInstance.external_fluid_source != Null:`  
```

Comment:
does 'not' need to be removed here? same comment as #495

Reply:
Agreed. That is a relic. Correct.